### PR TITLE
[1LP][RFR] - Removing duplicate test

### PR DESCRIPTION
**Pull request by dgaikwad**
Removed duplicate test case

### CODE REVIEW

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -2816,7 +2815,11 @@ def test_self_service_users_ssui():
             1. User created successfully
             2. SSUI access granted
     """
-    pass
+    user, _ = user_self_service_role
+
+    with user:
+        with appliance.context.use(ViaSSUI):
+            appliance.server.login(user)
```
Comment (valaparthvi) [inline cfme/tests/configure/test_access_control.py:2822]: It kinda looks like a smoke test. If you think so too, better to add a smoke test marker.

**Comment by niyazRedhat** [comment]
We already have automation covered for this senario

**Comment by john-dupuy** [comment]
@dgaikwad This could be easily parametrized and combined with !10105, but as @niyazRedhat if we already have coverage for this scenario we should just remove this test case. 

@niyazRedhat can you provide a link?

**Comment by dgaikwad** [comment]
@niyazRedhat Thanks for your suggestion. @john-dupuy found this test which is doing same: https://github.com/ManageIQ/integration_tests/blob/f6f0b52715fd9a0046af7689a0d9f5a2a83cae7b/cfme/tests/automate/custom_button/test_service_objects.py#L474

**Pull request merged**